libs/null.liq

```diff
@@ -0,0 +1,5 @@
+# Determine whether a nullable value is null.
+# @category Liquidsoap
+def null.is(x) =
```

Comment:
I think a better name would be `null.check`

Reply:
![giphy (1)](https://user-images.githubusercontent.com/871060/85044206-ee335e80-b152-11ea-8ac1-7e90e86a9a5a.gif)